web/themes/custom/recipe_theme/src/sass/base/_typography.scss

```diff
@@ -0,0 +1,5 @@
+@font-face {
+  font-family: 'Dosis';
+  src: url('../fonts/Dosis/Dosis-Regular.ttf') format('ttf'), url('../fonts/Dosis/Dosis-Regular.ttf') format('ttf');
```

Comment:
Gabor already included Dosis fonts in a different PR in a different file https://github.com/Pronovix/food/pull/45#discussion_r413958827

Please discuss who is going to add the font faces in the end so you don't have duplicates as you added the `font-face` in the `typography` file and Gabor added them in a `font_faces` file.

Reply:
I thought it's connected to my task (Implement design according to UI kit), but as Gábor is closer to merge his PR, I will delete my fonts and pull from master later.